python/core/qgsdataitem.sip

```diff
@@ -1,28 +1,46 @@
-/** Base class for all items in the model.
- *  Parent/children hierarchy is not based on QObject. */
+/************************************************************************
+ * This file has been generated automatically from                      *
+ *                                                                      *
+ * src/core/qgsdataitem.h                                               *
+ *                                                                      *
+ * Do not edit manually ! Edit header and run scripts/sipify.pl again   *
+ ************************************************************************/
+
+
+
+
+
+
+typedef QgsDataItem *dataItem_t( QString, QgsDataItem * );
```

Comment:
This isn't needed in python:

    typedef QgsDataItem *dataItem_t( QString, QgsDataItem * ) SIP_SKIP;